packages/components/carousel/src/main.vue

```diff
@@ -357,6 +357,17 @@ export default defineComponent({
 
     // lifecycle
     onMounted(() => {
+      const stopWatchItems = watch(
+        () => items.value,
+        (val) => {
+          if (val.length) {
+            playSlides()
+            stopWatchItems?.()
+          }
+        },
+        { immediate: true, deep: true }
+      )
+
```

Comment:
@freedomlang Did you accidentally included code which doesn't belong to this issue? Would you please remove them?

Reply:
Removed